js/src/logger.ts

```diff
@@ -2714,6 +2714,10 @@ export class ReadonlyExperiment extends ObjectFetcher<ExperimentEvent> {
 
 let executionCounter = 0;
 
+export function newId() {
```

Comment:
What's the point of this?

Reply:
Just factor this logic out so I don't need to remember how we create a uuid each place we need to create an id.